soh/soh/Enhancements/randomizer/ShuffleFairies.h

```diff
@@ -10,4 +10,4 @@ typedef struct FairyIdentity {
 } FairyIdentity;
 
 void ShuffleFairies_RegisterHooks();
-void ShuffleFairies_UnregisterHooks();
+void ShuffleFairies_UnregisterHooks();
```

Comment:
is moving these over to using `COND_HOOK` in a shipinit func something you were wanting to do in this PR, or would you want to do that in a second pass?

Reply:
Hmmm not actually sure why this is in the diff. Probably later, the goal here was splitting location list to avoid a stack overflow, ship init just seemed like the right way to accomplish that. Updating everything else to the proper way can wait until we're not in bugfix mode.